packages/PieChart/src/PieChart.test.tsx

```diff
@@ -0,0 +1,75 @@
+/**
+  * @jest-environment jsdom
+  */
+ import React from 'react';
+ import { render, screen } from '@testing-library/react';
+
+ import PieChart from './PieChart';
+
+ const setup = (props = {}) => {
+  return render(
+    <PieChart dataTest="ids-pie-chart" {...props} />
+  );
+};
+
+ describe('PieChart', () => {
+   test('It should render without errors', () => {
+    setup();
+     const wrapper = screen.getByTestId('ids-pie-chart');
+     expect(wrapper).toBeInTheDocument();
+   });
+
+   test('It should render a snapshot', () => {
+     const {asFragment} = setup();
+     expect(asFragment()).toMatchSnapshot();
+   });
+
+   test('It should render the provided data elements', () => {
+    const data = [
+      { id: 'data1', percentage: 30 },
+      { id: 'data2', percentage: 20 },
+      { id: 'data3', percentage: 50 },
+    ];
```

Comment:
Coud be nice to also test when the `sum > 100` and `sum < 100`

Reply:
I'm not sure how this benefits us in the tests? The data will be the same, it's not like we'll be testing the user data.
Also, recharts will always fill the chart. For example, if it adds up to only 60, it thinks the total is 60 and displays it accordingly. I guess it's up to the user to put the proper values. Unless you have another suggestion?

Reply:
Thinking back on it, I was thinking more of integration tests then unit tests, where we will validate that `recharts` will always fill the chart.

Overall my comment is not really relevant since we don't test integrations.